contentmgr/replication.go

```diff
@@ -297,17 +251,33 @@ func (cm *ContentManager) ToCheck(contID uint) {
 	}
 }
 
+func (cm *ContentManager) getReadyStagingZones() ([]util.Content, error) {
+	var readyZones []util.Content
+	if err := cm.DB.Model(&util.Content{}).
+		Where("not active and pinning and aggregate and size >= ?", constants.MinDealContentSize).
```

Comment:
do we need to add a filter to only get content that are relatively new? created within 3 months?

Reply:
no, this should be fine, it has nothing to do with pinning, it's basically just taking contents ready for deal making